### PR TITLE
Chapter2/homework/2 Max

### PR DESCRIPTION
**Pull request by HellHunterMax**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Inventory.cs  4
- Src/BootCamp.Chapter/Shop.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522&bid=16910169)

File: Src/BootCamp.Chapter/Player.cs [hunk 1]
```diff
@@ -12,50 +12,58 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Player
     {
-        /// <summary>
-        /// Everyone can carry this much weight at least.
-        /// </summary>
-        private const int baseCarryWeight = 30;
+        private void BonusTask()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Player.cs]: Codacy Issue found: [Remove the unused private method 'BonusTask'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)

File: Src/BootCamp.Chapter/Player.cs [hunk 2]
```diff
@@ -12,50 +12,58 @@ namespace BootCamp.Chapter
...
+        {
 
-        private string _name;
-        private int _hp;
+            /// <summary>
+            /// Everyone can carry this much weight at least.
+            /// </summary>
+            //private const int baseCarryWeight = 30;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Player.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)

File: Src/BootCamp.Chapter/Item.cs [hunk 3]
```diff
@@ -15,6 +15,10 @@ public decimal GetPrice()
         }
 
         private float _weight;
+        public float GetWeight()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Item.cs]: Codacy Issue found: [Consider making method 'GetWeight' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)

File: Src/BootCamp.Chapter/Shop.cs [hunk 4]
```diff
@@ -64,6 +75,17 @@ public decimal Buy(Item item)
         /// </returns>
         public Item Sell(string item)
         {
+            //TODO Sell- adds money to the shop. Shop can only sell items that it has.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Shop.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)

Comment (HellHunterMax) [reply Src/BootCamp.Chapter/Shop.cs]: fixed this in my new local build will upload as soon as reviewed.

File: Src/BootCamp.Chapter/Inventory.cs [hunk 5]
```diff
@@ -5,22 +5,66 @@ public class Inventory
...
+            int number = 0;
+            int[] placeOfItemsInArr = new int[_items.Length];
+
+            for (int i = 0; i < _items.Length; i++)
+            {
+                if (_items[i] != null && _items[i].GetName() == name)
+                {
+                    placeOfItemsInArr[number] = i;
+                    number++;
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Inventory.cs]: Please remove the redundant blank line.

**Pull request closed without merging**